salt/orch/etcd-migrate.sls

```diff
@@ -0,0 +1,155 @@
+# To migrate etcd storage version for Kubernestes execute following steps on Admin node:
+# 1. docker exec -ti $(docker ps | grep salt-master | cut -d" " -f1) salt-run state.orchestrate orch.etcd-migrate
+# 2. btrfs property set -ts /.snapshots/1/snapshot ro false
+# 3. mount -o remount,rw /
+# 4. sed -i '/etcd_version/ s/etcd2/etcd3/' /usr/share/salt/kubernetes/pillar/params.sls
+# 5. docker exec -ti $(docker ps | grep salt-master | cut -d" " -f1) salt '*' saltutil.refresh_pillar
+# 6. docker exec -ti $(docker ps | grep salt-master | cut -d" " -f1) salt '*' salt.highstate
```

Comment:
These three steps should be included within this migration orchestration. (For #4 - that should target the admin node, and set the pillar value to etcd3 in the database - probably with a `docker exec` again the velum container?)

Reply:
Ok, I will investigate how to update pillar value in database backend